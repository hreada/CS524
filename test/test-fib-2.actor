app(rec(lambda(fib, lambda(n,
      if(le(n, 2), 1, add(app(fib, minus(n, 1)),
                          app(fib, minus(n, 2))))))),
    7)
