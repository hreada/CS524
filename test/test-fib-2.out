<T>
  <actors>
    <actor>
      <k> exec ( 13 ) </k>
      <id> initactor </id>
    </actor>
  </actors>
  <messages> .MessageCellBag </messages>
  <definedaddr> ListItem ( initactor ) </definedaddr>
</T>
